src/components/UI/Form/AutoComplete/AutoComplete.tsx

```diff
@@ -261,7 +261,28 @@ export const AutoComplete = ({
                 {...textFieldProps}
                 data-testid="AutocompleteInput"
                 placeholder={placeholder}
-                sx={inputSxStyle}
+                sx={
+                  isFilterType
+                    ? {
+                        '& .MuiOutlinedInput-root': {
+                          height: '100%',
+                          paddingBottom: 0,
+                          paddingTop: 0,
+                        },
+                        '& fieldset': {
+                          borderRadius: '12px',
+                          border: 'none',
+                        },
+                        height: '100%',
+                        borderRadius: '10px !important',
+                        borderColor: '#93a29b',
```

Comment:
Create a object for styles and add here. The object should be outside of autocomplete component

Reply:
Done